### PR TITLE
Workflows holdingpen changes

### PR DESCRIPTION
**Pull request by jalavik**
This branch implements legacy branches to OAI harvest that the upcoming workflow changes from Guillaume is dependent on. So to avoid sending you a mega branch of 20 commits of different forms I felt like sending you these changes first that are from "the past" meant for master.

The suggested plan then is to move oai_harvest to the new structure after Guillaume's changes in the workflows module is integrated.

Please let us know if we are doing this the wrong way.

### CODE REVIEW

File: modules/miscutil/lib/upgrades/invenio_2012_12_05_oaiHARVEST_arguments_blob.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+from invenio.dbquery import run_sql, \
```
Comment (jirikuncar) [inline modules/miscutil/lib/upgrades/invenio_2012_12_05_oaiHARVEST_arguments_blob.py]: `from invenio.legacy.dbquery`

File: modules/oaiharvest/bin/oaiharvest.in [hunk 2]
```diff
@@ -44,6 +44,7 @@ Examples:
```
Comment (jirikuncar) [inline modules/oaiharvest/bin/oaiharvest.in:44]:
```
git mv modules/oaiharvest/bin/oaiharvest.in invenio/legacy/oaiharvest/scripts/oaiharvest.py
vim invenio/legacy/oaiharvest/scripts/oaiharvest.py # add @with_app_context to main function
vim setup.py # add this script's main function
```

**Comment by jalavik** [comment]
Fixes done. 

However, many more changes related to migration to new structure is coming with more commits from Guillaume who will move the files to the new structure and integrate workflows to it.

**Comment by jirikuncar** [comment]
Can you mark commits that should be later removed because they will come from master?

**Pull request closed without merging**